cnd/src/http_api/routes/post.rs

```diff
@@ -0,0 +1,161 @@
+use crate::{
+    asset,
+    created_swap::{CreatedSwap, HbitHerc20CreatedSwap, Herc20HbitCreatedSwap},
+    db::Save,
+    http_api::{problem, routes::into_rejection, Http},
+    identity,
+    network::{DialInformation, InitCommunication},
+    swap_protocols::{hbit, herc20, Facade, LocalSwapId, Role},
+};
+use serde::Deserialize;
+use warp::{http::StatusCode, Rejection, Reply};
+
+/// POST endpoints for the hbit/herc20 protocol pair.
```

Comment:
❓ Instead of having to clarify with this documentation string, should we just call the file something like `post_hbit_herc20.rs`. I guess eventually this all moves back to `index.rs` anyway.

Reply:
❓ I don't immediately see why all this is not in `index.rs` already anyway. How does this duplication help us?

Reply:
> Instead of having to clarify with this documentation string, should we just call the file something like post_hbit_herc20.rs.

I imagine _all_ the POST funtions to be in here.  This comment, that risks going stale, was to make it apparent that so far we only have this trading pair done - it feels like the `http_api/` is going to be hard to navigate because the naming of rfc003 files (`index.rs` etc.) is not being used now so its hard to tell which files are which.

> I don't immediately see why all this is not in index.rs already anyway. How does this duplication help us?

`index.rs` is going away, if I get my way, for 2 reasons. 1. because its not a useful name 2. Because it mostly contains halight code that needs rewriting. I did not want to break the LN code though so did not delete it yet.